src/src/PluginStructs/P099_data_struct.cpp

```diff
@@ -262,7 +279,7 @@ void P099_data_struct::scaleRawToCalibrated(uint16_t &x, uint16_t &y) {
       if (_y > StoredSettings.Calibration.bottom_right.y) {
         _y = StoredSettings.Calibration.bottom_right.y;
       }
-      float y_fact = (StoredSettings.Calibration.bottom_right.y - StoredSettings.Calibration.top_left.y) / ts_y_res;
+      float y_fact = (StoredSettings.Calibration.bottom_right.y - StoredSettings.Calibration.top_left.y) / _ts_y_res;

```

Comment:
Why store it in a float here?
All values are int types and not casted to a float, so it will only be converted to a float when assigning it, not during computation.

Reply:
Think I'm spoiled by all the Java programming I did in the past 🤔 will investigate and fix.